### PR TITLE
Clearing gradle warnings

### PR DESCRIPTION
**Pull request by AlexandrouR**
### What does this PR do?
This PR aims to fix Gradle warnings.
### Where should the reviewer start?
*required*

### Why is it needed?
*required*

### CODE REVIEW

File: core/src/main/java/org/web3j/tx/gas/DefaultGasProvider.java [hunk 1]
```diff
@@ -14,12 +14,9 @@
 
 import java.math.BigInteger;
 
-import org.web3j.tx.Contract;
-import org.web3j.tx.ManagedTransaction;
-
 public class DefaultGasProvider extends StaticGasProvider {
-    public static final BigInteger GAS_LIMIT = Contract.GAS_LIMIT;
-    public static final BigInteger GAS_PRICE = ManagedTransaction.GAS_PRICE;
+    public static final BigInteger GAS_LIMIT = BigInteger.valueOf(4_300_000);
```
Comment (cfelde) [inline core/src/main/java/org/web3j/tx/gas/DefaultGasProvider.java]: Should we make the default closer to 9_000_000 or 10_000_000, as that seems to be closer to the block limit these days..?

Comment (AlexandrouR) [reply core/src/main/java/org/web3j/tx/gas/DefaultGasProvider.java]: I think that is a good idea.

File: abi/src/test/java/org/web3j/abi/TypeDecoderTest.java [hunk 2]
```diff
@@ -348,7 +348,7 @@ public void testStaticArray() throws Exception {
         StaticArray2 arr =
                 (StaticArray2)
                         TypeDecoder.instantiateType("uint256[2]", new long[] {10, Long.MAX_VALUE});
-        assert (arr instanceof StaticArray2);
+        assert (arr != null);
```
Comment (antonydenyer) [inline abi/src/test/java/org/web3j/abi/TypeDecoderTest.java]: Are you sure about these, you probably want something like
`assertThat(arr, instanceOf(StaticArray2.class));`

File: abi/src/test/java/org/web3j/abi/TypeDecoderTest.java [hunk 3]
```diff
@@ -348,7 +348,7 @@ public void testStaticArray() throws Exception {
...
```
Comment (AlexandrouR) [inline abi/src/test/java/org/web3j/abi/TypeDecoderTest.java]: replace with assertThat(arr, instanceOf(StaticArray2.class));

File: abi/src/test/java/org/web3j/abi/TypeDecoderTest.java [hunk 4]
```diff
@@ -414,13 +414,14 @@ public void testDynamicArray() throws Exception {
                 (DynamicArray)
                         TypeDecoder.instantiateType(
                                 "string[]", new String[] {"Hello, world!", "world! Hello,"});
-        assert (arr instanceof DynamicArray);
+        assert (arr != null);
```
Comment (AlexandrouR) [inline abi/src/test/java/org/web3j/abi/TypeDecoderTest.java]: replace with assertThat(arr, instanceOf(StaticArray2.class));

File: codegen/src/main/java/org/web3j/codegen/TupleGenerator.java [hunk 5]
```diff
@@ -54,7 +54,7 @@ private void generate(String destinationDir) throws IOException {
     }
 
     private TypeSpec createTuple(int size) {
-        String javadoc = "@deprecated use 'component$L' method instead";
+        String javadoc = "@deprecated use 'component$L' method instead @return returns a value";
```
Comment (AlexandrouR) [inline codegen/src/main/java/org/web3j/codegen/TupleGenerator.java]: add \n

File: core/src/main/java/org/web3j/tx/gas/DefaultGasProvider.java [hunk 6]
```diff
@@ -14,12 +14,9 @@
...
+    public static final BigInteger GAS_PRICE = BigInteger.valueOf(22_000_000_000L);
```
Comment (AlexandrouR) [inline core/src/main/java/org/web3j/tx/gas/DefaultGasProvider.java]: replace with 4_100_000_000L

File: integration-tests/src/test/java/org/web3j/protocol/scenarios/HumanStandardTokenGeneratedIT.java [hunk 7]
```diff
@@ -17,34 +17,36 @@
...
 import static org.web3j.generated.HumanStandardToken.ApprovalEventResponse;
 import static org.web3j.generated.HumanStandardToken.TransferEventResponse;
-import static org.web3j.generated.HumanStandardToken.deploy;
 import static org.web3j.tx.TransactionManager.DEFAULT_POLLING_FREQUENCY;
 
 /** Generated HumanStandardToken integration test for all supported scenarios. */
 public class HumanStandardTokenGeneratedIT extends Scenario {
 
     @Test
+    @Ignore
```
Comment (AlexandrouR) [inline integration-tests/src/test/java/org/web3j/protocol/scenarios/HumanStandardTokenGeneratedIT.java]: Wrong ignore import

File: integration-tests/src/test/java/org/web3j/protocol/scenarios/SimpleStorageContractIT.java [hunk 8]
```diff
@@ -14,20 +14,25 @@
...
+import org.web3j.tx.gas.ContractGasProvider;
+import org.web3j.tx.gas.DefaultGasProvider;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 public class SimpleStorageContractIT extends Scenario {
 
     @Test
+    @Ignore
```
Comment (AlexandrouR) [inline integration-tests/src/test/java/org/web3j/protocol/scenarios/SimpleStorageContractIT.java]: Wrong Ignore

File: crypto/src/main/java/org/web3j/crypto/TransactionEncoder.java [hunk 9]
```diff
@@ -51,12 +51,6 @@
         return encode(rawTransaction, eip155SignatureData);
     }
 
-    @Deprecated
```
Comment (AlexandrouR) [inline crypto/src/main/java/org/web3j/crypto/TransactionEncoder.java]: Put this back

**Pull request closed without merging**